### ISSUE TITLE
Deprecation warning with Rails 6.1.0-alpha

### ISSUE THREAD

**Issue by sedubois** [newrelic/rpm#330]
```rb
DEPRECATION WARNING: [] is deprecated and will be removed from Rails 6.2 (Use configs_for) (called from block in <class:EnvironmentReport> at ~/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/newrelic_rpm-6.10.0.364/lib/new_relic/environment_report.rb:73)
```

Tested with rails master (revision 9c4171bae14db25f6c663c1f52b850ff684ba37d) and newrelic_rpm 6.10.0.364.

**Comment by rachelleahklein** [comment]
Thank you for bringing this to our attention, @sedubois. It will be good to have in mind when we are working on official Rails 6.1 support.

**Comment by sedubois** [comment]
Hi @rachelleahklein, happy to help. I've proposed a fix in #331. Could you consider already fixing this deprecation for those of us who are already working with Rails 6.1 (whether it be to support its preparation or to benefit early from its features)?

**Comment by rachelleahklein** [comment]
Hi @sedubois, thank you for submitting a PR to fix this! We'll take a look, but I don't foresee any issue with including the fix in our next release. We appreciate your work!

**Comment by tannalynn** [comment]
This has been fixed in our newest release, version 6.11.0.
Thank you!
